docs/configuration/http_filters/fault_filter.rst

```diff
@@ -66,6 +68,20 @@ upstream_cluster:
   the config are present in the request with the same values (or based on presence if the ``value``
   field is not in the config).
 
+.. _config_http_filters_fault_injection_downstream_cluster:
+
+downstream_cluster_specific_settings:
```

Comment:
In general it seems odd to me that this needs to be specified at all. I would just always look for downstream cluster variants if the header is set, and use the default settings as the defaults for the lookups. Is there any particular reason to have this be configurable separately?